patches/chrome-browser-extensions-extension_service.h.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
-index 1265719e1b83f08e79827e871e13e15883aea508..aeb9809eea27f8d249c8737a179ca29b73b9afdd 100644
+index 1265719e1b83f08e79827e871e13e15883aea508..c6325944935e4f7d11b3fff0342e0e3bda472df6 100644
 --- a/chrome/browser/extensions/extension_service.h
 +++ b/chrome/browser/extensions/extension_service.h
-@@ -491,6 +491,7 @@ class ExtensionService : public ExtensionServiceInterface,
+@@ -490,6 +490,7 @@ class ExtensionService : public ExtensionServiceInterface,
+   // While disabled all calls to CheckForExternalUpdates() will bail out.
    static base::AutoReset<bool> DisableExternalUpdatesForTesting();
  
++  BRAVE_EXTENSION_SERVICE_H
```

Comment:
do not automatically replace these with another patch. There is often a `define` we can use like the other ones I flagged